ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Public/AccessibilityStatement.cshtml

```diff
@@ -9,88 +9,111 @@
-			The Department of Education (DfE) is committed to making its website accessible, in accordance with the
-			Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+			This website is partially compliant with the WCAG version 2.1 AA standard, due to the following non-compliance(s) and exemptions. 
 		</p>
 
-		<h3 class="govuk-heading-m">Compliance status</h3>
+		<h3 class="govuk-heading-m">Non-accessible content </h3>
+		<p class="govuk-body">
+			The content is non-accessible for the following reason(s): 
+		</p>
+		<h4 class="govuk-heading-s">
+		Non-compliance with the accessibility regulations
+		</h4>
+		<ul class="govuk-list govuk-list--bullet">
+			<li>keyboard users cannot select the save and continue button on conversions project pages. This fails WCAG 2.1 AA success criteria 2.1 (Keyboard Accessible) </li>
+			<li>the website tells JAWS users there are form fields present where there are none. This fails WCAG 2.1 AA success criteria 2.4.4 (Link Purpose (In Context)) </li>
+			<li>main headings on some pages are contained within groups of elements and so may not be read out correctly by some screen reader software. This fails WCAG 2.1 AA success criteria 2.4.6 (Headings and Labels)</li>
+			<li>some labelling on primary action buttons are not consistent. The labels were selected to give users a better description of the purpose of the action based on user research. This fails WCAG 2.1 AA success criteria 3.2.4 (Consistent Identification) </li>
+			<li>when information is deleted from a required field, this is not announced when the page is reloaded. Users must navigate down the page to find the error message. This fails WCAG 2.1 AA success criteria 3.3.1 (Error Identification) </li>
+			<li>when users start a new transfer project and do not select an outgoing trust, the error message is not clear about what users should do. This fails WCAG 2.1 AA success criteria 3.3.1 (Error Identification) </li>
+		</ul>
+
+		<h4 class="govuk-heading-s">
+			Disproportionate burden
+		</h4>
 		<p class="govuk-body">
-			This website complies with the Web Content Accessibility Guidelines version 2.1 AA standard.
+			We have determined that there is disproportionate burden to meet all the requirements. Making improvements would leave us unable to provide a minimum viable product (MVP). We will meet the requirements as soon as we can. 
 		</p>
+		<h4 class="govuk-heading-s">
+			The content is not within the scope of this accessibility statement
+		</h4>
+		<ul class="govuk-list govuk-list--bullet">
+			<li>data that is populated into our website from <a class="govuk-link" href="https://apply-convert-academy.service.gov.uk/">Apply to become an academy</a> and other Department for Education websites </li>
+			<li>content hosted on other Department for Education websites, such as <a class="govuk-link" href="https://apply-convert-academy.service.gov.uk/">Apply to become an academy</a></li>
```

Comment:
These links are for production. Is that an issue for staging? & dev?

Reply:
I think it's fine. It's an external link so I don't think we need a staging or dev link for it.